src/projectApplications.js

```diff
@@ -0,0 +1,137 @@
+const fs = require('fs');
+const path = require('path');
+const {google} = require('googleapis');
+
+const SCOPES = ['https://www.googleapis.com/auth/spreadsheets.readonly'];
+const ALL_QUARTERS_ID = '16vmuNiN-8mij8EoEIwvQ_wde9DgF_0IWXfT_95SHfcc';
+
+const numOfQuarters = 14;
+
+// Load client secrets from a service account
+fs.readFile(path.resolve(__dirname, '../credentials.json'), (err, content) => {
+  authorize(JSON.parse(content), main);
+});
+
+function authorize(credentials, callback) {
+  const jwt = new google.auth.JWT(credentials.client_email, null, credentials.private_key, SCOPES);
+  callback(jwt);
+}
+
+function main(auth) {
+  process.stdout.write('Parsing project sheet .. ');
+  readProjectMemberSheet(auth);
+}
+
+function readProjectMemberSheet(auth) {
+  const sheets = google.sheets({version: 'v4', auth});
+  const endQuarter = ['Winter', '2021'];
+  const startQuarter = ['Fall', '2016'];
+  const quarters = ['Winter', 'Spring', 'Fall'];
+  let current = startQuarter;
+  let masterDict = [];
+  let count = 0;
+
+  
+  while(current[1] < endQuarter[1] ||
+    (current[1] == endQuarter[1] && quarters.indexOf(current[0]) <= quarters.indexOf(endQuarter[0]))){
+    
+    const currentQuarter = current[0];
+    const currentYear = current[1];
+    // Run on current quarter
+    sheets.spreadsheets.values.get({
+      spreadsheetId: ALL_QUARTERS_ID,
+      range: `${current[0]} ${current[1]}!A:Z`,
+    }).then((response) => {
+      const info = formatProjectMemberResponse(response);
+      // console.log(info);
+      let memberinfo = [];
+      let quarter = [];
+      for(var i = 0; i<info.length; i++) {
+        if(!(info[i][0] in masterDict)) {
+          //console.log('running if statement');
+          
+          quarter = new Array(numOfQuarters).fill(0);
+          quarter[count] = 1;
+          memberinfo = [info[i][1], info[i][2], quarter];
+          //console.log(memberinfo);
+          masterDict.push({
+            key: info[i][0],
+            value: memberinfo,
+          });
+        }
+        else
+        {
+          //console.log('run else statement');
+          // masterDict[info[i][0]][2][count] = 1;
+          quarter[count] = 1;
+        }
+        if (info[i][3]==2)
+        {
+          quarter[count] = 2;
+        }
+      }
+      count++;
+      console.log(count);
+      fs.writeFileSync(path.resolve(__dirname, '../data/projectmember.json'), JSON.stringify(masterDict));
```

Comment:
```suggestion
      fs.writeFileSync(path.resolve(__dirname, '../data/projectmember.json'), JSON.stringify(masterDict, null, 2));
```
just to make the output look prettier